etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiResult.java

```diff
@@ -1,24 +1,78 @@
 package gov.hhs.cdc.trustedintermediary.external.hapi;
 
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.etor.messages.MessageHdDataType;
 import gov.hhs.cdc.trustedintermediary.etor.results.Result;
+import java.util.function.Supplier;
 import org.hl7.fhir.r4.model.Bundle;
 
 /** Filler concrete implementation of a {@link Result} using the Hapi FHIR library */
 public class HapiResult implements Result<Bundle> {
 
+    private final HapiMessageHelper MESSAGE_HELPER =
```

Comment:
Same comment as above

Reply:
Also, seems like there is a lot of shared code between `HapiOrder` and `HapiResult`

Reply:
Same...
No. Conventionally, final variables are all uppercase and use underscore. Sonarcloud is just angry and wants us to follow the camel case convention. I think we can leave it as is, thoughts?

Reply:
Yes on the similar codes. There is a Message interface that both Result and Order interfaces extend. HapiOrder implements Order and HapiResult implements Result. We can discuss if we no longer need HapiOrder or HapiResult and just create HapiMessage to do the work. But we already have two separations for Orders and Results., thoughts?

Reply:
I know there are some differences between HapiOrder and HapirResult so they may not be able to completely merge

Reply:
Checking what we can merge by having a common parent.